.changeset/stale-rocks-press.md

```diff
@@ -0,0 +1,5 @@
+---
+"@atproto/aws": patch
+---
+
+Set a 5 seconds timeout on every `S3BlobStore` requests
```

Comment:
We should scrutinize whether we like this timeout, as sometimes are are working with somewhat large files (e.g. videos).

Reply:
File uploads are still using a larger (configurable) timer (which defaults to 10 seconds)